notifications/slack.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# Post to Slack channel on new deployment
+# https://api.slack.com/incoming-webhooks
+#
+# You can either add those here, or configure them on the environment tab of your
+# project settings.
+SLACK_WEBHOOK_TOKEN="T02CLSA7U/B051FFP0B/jtRHTrx7rp2un5rS3gsmZvKQ"
```

Comment:
You very likely want to **immediately** reset this webhook integration, or anybody can send messages to your channel!

Please never include tokens or other methods to authenticate with ressources in the PRs!